README.md

```diff
@@ -792,6 +792,15 @@ backends:
                 image: ubuntu:16.04.1
 ```
 
+Additionally, you can provide a comma-seperated list of LXD profiles to use:
+```
+backends:
+    lxd:
+        profiles: my-profile
```

Comment:
probably this setting should be more specifically named, maybe `container-profiles` ?

Reply:
This is a good point, it would avoid any ambigitity with other backends in the future.